iotkit/helloworld/README.md

```diff
@@ -0,0 +1,59 @@
+# Hello World
+
+This document summarizes how to build, deploy and run the helloworld
+program with a GCP IoT Kit. Once you are finished, you will have a display
+blinking "Hello gopher!" with a blue backlight.
+
+![Hello gopher!](https://i.imgur.com/TKI5Iz1.gif)
+
+## Building
+
+First, go get the sample program.
+
+    $ go get -d github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
+
+Then, build a binary targeting the GCP IoT Kit board, BeagleBone Green
+Wireless, by running the command below.
+
+    $ GOOS=linux GOARCH=arm GOARM=7 go build github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
+
+The command below will generate a binary that can executable on your board.
+
+## Setting up the board
+
+Download [Debian 8.5 2016-05-13 4GB SD LXQT](https://beagleboard.org/latest-images)
+and write it to an microSD card.
+
+Put the microSD card into the SDcard slot and power the board to boot.
+Once the board boots, you will be able to see an unsecured Wifi network
+in the format of "BeagleBoneXXXX".
+
+Join to the "BeagleBoneXXXX" network from your laptop and visit
+[192.168.8.1:3000/ide.html](192.168.8.1:3000/ide.html) to launch the
```

Comment:
Add http:// to the URL. It didn't look linkable here:
https://github.com/GoogleCloudPlatform/golang-samples/blob/141b173893f1f7a855547d7ad0f1ea01488cc426/iotkit/helloworld/README.md